bin/console

```diff
@@ -4,13 +4,25 @@
 declare(strict_types=1);
 
 use Antidot\Cli\Application\Console;
+use Antidot\Framework\Runtime\AntidotRuntime;
+
+ini_set('memory_limit', '2048M');
```

Comment:
Is this memory setting configurable?

Reply:
Yes I put 2048M because when hitting a stress test with concurrency it reaches the default memory limit, it should be configurable by framework config too

Reply:
take a look to https://github.com/the-benchmarker/web-frameworks/pull/4640